sceval_frontend/src/containers/SensorModule.tsx

```diff
@@ -1054,6 +1055,7 @@ export const SensorModule = withRouter((props: SensorModuleProps & RouteComponen
         fetchDetailedData(masterDateBounds);
       }
     }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [realtimeMode, masterDateBounds]);
```

Comment:
This also has a missing dependencies warning but it can't be fixed so we will suppress the warning instead.